### ISSUE TITLE
Probléme d'installation

### ISSUE THREAD

**Issue by PatrickMag** [Laurux/Laurux#93]
Erreur remontée du forum : https://www.laurux.fr/drupal/node/177#comment-414
J'ai installé un poste sur ubuntu aujourd'hui, j'en ai donc profité pour installer laurux 3.681 et j'ai eu la même erreur.

**Comment by damscot** [comment]
la creation de la base à partir de la fenetre de connection est supporté
il n'y a que la creation de l'utilisateur qu'il faut faire a coté...
j'ai raté quelque chose ?


2018-03-14 21:48 GMT+01:00 PatrickMag <notifications@github.com>:

**Comment by PatrickMag** [comment]
Non,  c'est la procédure indiquée sur la doc.  Je suppose que Jack voulait vérifier les droits de l'utilisateur.
J'ai voulu faire installer gambas par le gestionnaire graphique et c'était une mauvaise idée

**Comment by damscot** [comment]
le pb de l'installateur graphique c'est qu'il n'installe pas qt4 et pas
gambas 3.10...
ceci dit le fait d'utiliser le ppa est aussi une mauvaise idee car on ne
maitrise pas le numero de version installé.
Il faudrait conserver une archive des packages sur le site internet de
Laurux
